app/src/main/java/org/oppia/app/testing/ImageRegionSelectionTestActivity.kt

```diff
@@ -0,0 +1,23 @@
+package org.oppia.app.testing
+
+import android.os.Bundle
+import org.oppia.app.activity.InjectableAppCompatActivity
+import org.oppia.app.utility.ClickableAreasImage
+import org.oppia.app.utility.OnClickableAreaClickedListener
+import javax.inject.Inject
+
+/** Test Activity used for testing [ClickableAreasImage] functionality */
+class ImageRegionSelectionTestActivity :
+  InjectableAppCompatActivity(),
+  OnClickableAreaClickedListener {
+
+  @Inject lateinit var imageRegionSelectionTestActivityPresenter: ImageRegionSelectionTestActivityPresenter // ktlint-disable max-line-length
```

Comment:
Yes it would be still too long

Reply:
Can you still put the @Inject at one line and the lateinit var.. in another since we should break away the parts of the statement that can be wrapped in different lines